tests/integration_test/backend_retry_test.go

```diff
@@ -203,10 +282,23 @@ func TestBackendRetry(t *testing.T) {
 
 			time.Sleep(time.Second * 5)
 			gotSpanNames, _ := s.FakeStackdriverServer.RetrieveSpanNames()
-			if !reflect.DeepEqual(gotSpanNames, tc.wantSpanNames) {
-				t.Errorf("got span names: %+q, want span names: %+q", gotSpanNames, tc.wantSpanNames)
+			realWantSpanNames := tc.wantSpanNames
+			if tc.backendTypeUsed == remoteBackend {
+				realWantSpanNames = replaceBackendPort(tc.wantSpanNames, strconv.Itoa(int(s.Ports().DynamicRoutingBackendPort)))
```

Comment:
Good idea. I think you need to add the same logic to `opencensus_tracing_test.go`. Probably best to put in some utility file.

Reply:
You mean the port replacement part?

Reply:
Yes, but nevermind. Turns out I added this logic there as well:

https://github.com/GoogleCloudPlatform/esp-v2/blob/9fbb88c56b3b7b7e14a43b64b77ca2d8e7d2a0e0/tests/integration_test/opencensus_tracing_test.go#L406

Reply:
Cool!